app/views/invoices/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with controller: :invoices %>
```

Comment:
Not sure about this. Why do you have a form here?

Reply:
Just checking if you're awake